components/BusinessCard.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <v-card class="business-card">
+    <!-- City -->
+    <v-chip class="px-2 business-city">
+      <span>{{ city }}</span>
+    </v-chip>
+    <!-- Image -->
+    <div class="business-image">
+      <img
+        :alt="business.title + ' image'"
```

Comment:
alt tag should describe the picture, business.title is fine, 'image' is redundant